2019-10-04-Ayush.md

```diff
@@ -0,0 +1,4 @@
+---
+name - "Ayush Patel"
+github_id: "in-ayushpatel"
+---
```

Comment:
Also add your description below this line. And make sure you add some appropriate description, as it will be shown on a live website.